Geometry/VeryForwardGeometry/python/geometryRPFromDB_cfi.py

```diff
@@ -1,12 +1,18 @@
 import FWCore.ParameterSet.Config as cms
+from Geometry.VeryForwardGeometryBuilder.CTPPSGeometryESModule_cfi import CTPPSGeometryESModule
```

Comment:
Here is the problem. You are telling the system to also make the name `CTPPSGeometryESModule` available in this python module. 

Instead, do either
```python
import Geometry.VeryForwardGeometryBuilder.CTPPSGeometryESModule_cfi
```
and then only the clone line do
```python
ctppsGeometryESModule = Geometry.VeryForwardGeometryBuilder.CTPPSGeometryESModule_cfi.CTPPSGeometryESModule.clone(
```

or alternatively 
```python
from Geometry.VeryForwardGeometryBuilder.CTPPSGeometryESModule_cfi import CTPPSGeometryESModule as _ CTPPSGeometryESModule
```
and then the clone line becomes
```python
ctppsGeometryESModule =_CTPPSGeometryESModule.clone(
```

Reply:
Oh, did not know about this "underscoring" trick, many thanks @Dr15Jones! Maybe we can proceed as it is in 02dbab4cd08b7ee2943d6d077d0aa25dd0d971b4 and leave this cleanup for a follow-up PR?

Reply:
I meant 497ac85204102fe68bd3fda34e7659058cca3715. Sorry, now ready for testing.